lib/screens/v2/widget_instance/alert.ex

```diff
@@ -17,30 +17,59 @@ defmodule Screens.V2.WidgetInstance.Alert do
           screen: Screens.Config.Screen.t(),
           alert: Alert.t(),
           stop_sequences: list(list(stop_id())),
-          routes_at_stop: list(%{route_id: route_id(), active?: boolean()})
+          routes_at_stop: list(%{route_id: route_id(), active?: boolean()}),
+          now: DateTime.t()
         }
 
-  def active?(%__MODULE__{alert: alert}, happening_now? \\ &Alert.happening_now?/1) do
-    happening_now?.(alert)
+  @base_priority 2
+
+  @effect_priorities ~w[shuttle stop_closure suspension station_closure detour stop_moved snow_route elevator_closure]a
```

Comment:
[might be just me getting a little lost in translation but] a quick question/suggestion: would it make sense to rename this to `@event_priorities` or `@alert_type_priorities`? (if I'm understanding this right). Please disregard if not.

Reply:
Those would also make sense, but the official terminology in alert data for these values is `effect`. E.g. `{..., "effect": "STOP_MOVE", ...}` for [this alert](https://api-v3.mbta.com/alerts/384443). For consistency's sake I tried to stick with the terminology used by alerts.